svs.tex

```diff
@@ -39,96 +39,90 @@ \subsection{Objectives}
+The Wide Survey is implemented using a configuration similar to the LSST Wide-Fast-Deep, but constrained to a region of $\pm10$~deg in ecliptic latitude.
+By concentrating the observations within this region, it is possible to build several hundred degrees of template coverage in multiple bands within the first month the SV survey, thus enabling survey-scale tests of Prompt Processing with difference image analysis during the SV surveys.
+The ecliptic region is selected to test Solar System Object processing pipelines at LSST survey scale.
+The planned footprint extends across the southern Galactic cap, touching the Galactic plane on both sides, thus covering regions with a range of stellar densities and stellar populations, as well as a large contiguous low-dust region at high Galactic latitude.
+The footprint spans a range of declinations, allowing access for other ground-based telescopes located in the southern and northern hemispheres.
+The footprint coincides with the low-dust WFD, Galactic plane, and North Ecliptic Spur from the standard LSST Cadence, with a ratio of visits and band coverage in each region matching that of the standard LSST cadence.
+For example, the band coverage in the low-dust WDF and Galactic plane is $ugrizy$, while the north ecliptic spur recieves a smaller total number of visits and limited to the $griz$ bands.
 
-%Observation Timeline (baseline):
-%2 weeks	Wide-area Science Validation Survey: Template Generation Phase
-%4 weeks	10-year Depth Science Validation Survey
-%2 weeks	Wide-area Science Validation Survey: Realtime Alert Production Phase
+\textit{Template Generation Survey}
 
-The Deep Science Validation Survey is designed to testing coadds at the equivalent LSST 10-year survey full depth and beyond.
-The Wide Science Validation Survey is designed to approximate the difference imaging templates and data rates that would be expected during the first year of LSST, thus also providing a sustained full-scale test of the Data Facility.
-%early science operations,
-The Scheduler will drive nighttime observatory operation during the Science Validation Surveys.
+The SV surveys are a Rubin Observatory Construction deliverable.
+While the primary objective of the SV surveys is to fulfill the Construction Completeness criteria defined in this document, a secondary goal of the SV surveys is to facilitate a smooth transition to operations, and where possible, enhance the impact of the early Science Program.
+Multiple science cases would benefit from acquiring wide-area template coverage during the on-sky commissioning period to enhance opportunities for time-domain science during the first year of LSST, including time-limited target-of-opportunity science cases that require contemporaneous observations with other facilities.
+Accordingly, Rubin Observatory is exploring the technical feasibility of such observations as part of the SV surveys on a best-effort basis.
```

Comment:
The "\begin{itemize}" list of 3 surveys above give the impression that this "template" survey is decided not "is exploring the technical feasibility of" -- I'd suggest removing it from the list of three above and only discussing the Deep and WFD-like surveys as the key components of the SV Surveys. The in a separate (sub)section, introduce this possible template survey

Reply:
It will say the same thing, just manage expectations better